source/content/partials/drupal/prepare-local-environment-no-clone-new.md

```diff
@@ -10,7 +10,7 @@ reviewed: ""
 
 1. Review the documentation on [Git](/guides/git/), [Composer](/guides/composer), and [Terminus](/terminus), and install and configure them on your local computer. Pantheon requires Composer 2, at minimum.
 
-   - Mac users can use [Homebrew](https://brew.sh/) to install Git, Composer, and PHP, along with their required dependencies. **Note:** Terminus 3 should be used for PHP >= 8.0. Restart the shell or terminal environment after entering the following command:
+   - Mac users can use [Homebrew](https://brew.sh/) to install Git, Composer, and PHP, along with their required dependencies. **Note:** Terminus 4 should be used for PHP >= 8.4. Restart the shell or terminal environment after entering the following command:
```

Comment:
ooooh, I read this line and thought that Terminus only worked with 8.4 and above. We should probably instead say something like "if you are using PHP 8.4 or higher you must use Terminus 4. If you are still running PHP 8.2 or 8.3 locally, you can still use Terminus 3, though we recommend upgrading to Terminus 4 prior to the sunsetting of support for Terminus 3 on [DATE] 2026.

We should rewrite this line. And then perhaps some of my earlier comments about needing more compatibility tables and additional headings are irrelevant.